res/openvpn-gui-res-pl.rc

```diff
@@ -165,20 +166,20 @@ BEGIN
     AUTOCHECKBOX "Uruchamiaj po zalogowaniu się użytkownika", ID_CHK_STARTUP, 17, 59, 200, 12
 
     GROUPBOX "Preferencje", ID_GROUPBOX3, 6, 82, 235, 165
-    AUTOCHECKBOX "Dodaj do dziennika", ID_CHK_LOG_APPEND, 17, 95, 60, 10
+    AUTOCHECKBOX "Dopis. do logów", ID_CHK_LOG_APPEND, 17, 95, 60, 10
```

Comment:
"Dopisz do logów" thats all we need (same string length, no abbreviation needed)

Reply:
Yes, but…

![Zrzut ekranu 2024-03-21 134558](https://github.com/OpenVPN/openvpn-gui/assets/1129853/26fb8a7b-2e54-4d08-93bd-c1e1766b8dcb)

And, yes, it is the same length, that is: the same character count, but not necessarily the same width.

I know we can change label geometry, but I didn't have time for experiments 😉